recipes-dotnet/corefx/corefx_1.1.0.bb

```diff
@@ -34,19 +43,34 @@ do_install() {
 
 	install -d ${target}
 
-	for i in `${src}/Linux.arm.Release/Native/*.so`
+	# Install the stripped binaries, the unstripped are smaller and causes Bus error
+	for i in `ls ${src}/Linux.arm.Release/Native/*.so`
 	do
-		# The non-stripped versions are called *.so.dbg [already-stripped]
-		install -m 0755 ${src}/Linux.arm.Release/Native/${i}.dbg ${target}/${i}
+		install -m 0755 ${i} ${target}
 	done
-	
+
 	for i in ${DLLS}
 	do
 		install -m 0644 ${src}/${i} ${target}
 	done
+
+	# Add link so the runtime can find libuv
+	ln -s /usr/lib/libuv.so.1 ${target}/libuv.so
+
+	# Add DLLs from Microsofts tools distribution
+	for i in ${TOOLS_DLLS}
+	do
+		install -m 0644 ${S}/Tools/${i} ${target}
+	done
 }
 
 FILES_${PN} = "/opt/dotnet"
+TOOLS_DLLS = "\
+  Microsoft.CodeAnalysis.CSharp.dll \
+  Microsoft.CodeAnalysis.dll \
+  Microsoft.CodeAnalysis.VisualBasic.dll \
+  Microsoft.CSharp.dll \
+"
 
```

Comment:
Correct me if I am wrong @ernstp, but I think we needed them to run aps.net core solutions using corerun. At this point we manage to run basic examples in mvc, websocket, static files using ssl. We have stripped out tooling-nugets that belongs to dotnet-cli capabilities however still *.CodeAnalysis.* was required.